sw/device/silicon_creator/rom/rom_start.S

```diff
@@ -156,20 +163,20 @@ _rom_start_boot:
   li t1, (1 << PWRMGR_CFG_CDC_SYNC_SYNC_BIT)
   sw t1, PWRMGR_CFG_CDC_SYNC_REG_OFFSET(t0)
 
-  // Setup the watchdog bite timer in order to reset the chip if the ROM stalls.
-  // The value below corresponds to 1 s for a clock frequency of 200 kHz.
-  // Since interrupts are disabled the watchdog bark will have no effect
-  // therefore the bark threshold is just set to the highest possible value.
+  // Configure the watchdog timers to reset the chip in the event that the ROM
+  // stalls. The watchdog's bark and bite thresholds are set to 1 and 16
+  // seconds, respectively, assuming a clock frequency of 200 kHz.
   li t0, TOP_EARLGREY_AON_TIMER_AON_BASE_ADDR
-  li t1, 0x30d40
-  sw t1, AON_TIMER_WDOG_BITE_THOLD_REG_OFFSET(t0)
-  li t1, 0xffffffff
+  li t1, 0x30d400
   sw t1, AON_TIMER_WDOG_BARK_THOLD_REG_OFFSET(t0)
+  li t1, 0x30d4000
```

Comment:
@alphan I also meant to ask if you think this bite threshold is outlandishly large. Initially, I only doubled it, but I think that was not sufficient to thread the needle with the debugger. Multiplying by 16 was the next thing I tried since it was easy, just adding a '0', so we might be able to tune it a little.

Reply:
Yes, it's a bit large, 1.1x should be OK I guess. Can you add the following definitions at the start of the file (maybe after the definition for `UNIMP`?
```
.equ WDOG_BITE_THOLD, 0x30d400
.equ WDOG_BARK_THOLD, WDOG_BITE_THOLD*11/10
```

Please move the comment about the bite threshold there as well.